lib/signs/utilities/sign_text.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Signs.Utilities.SignText do
+  @moduledoc """
+  Functions for handling Sign Text Length information
+  """
```

Comment:
nit: I'm not sure this needs a whole new module, I'd probably just toss these in `PaEss.Utilities`